<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerTrackerStabilizerModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerTrackerStabilizerModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Input</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Output</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="qMRMLNodeComboBox" name="InputTransformWidget">
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLLinearTransformNode</string>
           </stringlist>
          </property>
          <property name="addEnabled">
           <bool>false</bool>
          </property>
          <property name="removeEnabled">
           <bool>false</bool>
          </property>
          <property name="renameEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="qMRMLNodeComboBox" name="OutputTransformWidget">
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLLinearTransformNode</string>
           </stringlist>
          </property>
          <property name="baseName">
           <string>TrackerStabilizerFiltering</string>
          </property>
          <property name="renameEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="FilteringBox">
        <property name="title">
         <string>Filter</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Smoother</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Faster</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="ctkDoubleSpinBox" name="DoubleSpinBox">
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <double>1.000000000000000</double>
             </property>
             <property name="maximum">
              <double>50.000000000000000</double>
             </property>
             <property name="value">
              <double>7.500000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="ctkDoubleSlider" name="FilteringValueWidget">
             <property name="value">
              <double>7.500000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.500000000000000</double>
             </property>
             <property name="minimum">
              <double>1.000000000000000</double>
             </property>
             <property name="maximum">
              <double>50.000000000000000</double>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="text">
      <string>Advanced</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Active module node:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="qMRMLNodeComboBox" name="ModuleNodeComboBox">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLTrackerStabilizerNode</string>
         </stringlist>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSlider</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSlider.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerTrackerStabilizerModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>InputTransformWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTrackerStabilizerModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>OutputTransformWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTrackerStabilizerModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ModuleNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FilteringValueWidget</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DoubleSpinBox</receiver>
   <slot>setValueIfDifferent(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FilteringValueWidget</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
